File: dotnet/Example/Sort/Inheritance/Animal.cs
namespace Example.Sort.Inheritance
{
    public class Animal
    {
        private Otterop.Lang.String word;
        public Animal(Otterop.Lang.String word)
        {
            this.word = word;
        }

        public Otterop.Lang.String Act()
        {
            return this.word;
        }

        public Otterop.Lang.String Species()
        {
            return Otterop.Lang.String.Wrap("unknown");
        }

    }

}

File: src/main/java/example/sort/inheritance/Animal.java
package example.sort.inheritance;

import example.sort.tree.Node;
import otterop.lang.String;

public class Animal {
    private String word;

    public Animal(String word) {
        this.word = word;
    }

    public String act() {
        return this.word;
    }

    public String species() {
        return String.wrap("unknown");
    }
}
